inductiva/structures/scenarios/plate_linear_elastic/bc_utils.py

```diff
@@ -0,0 +1,105 @@
+"""Utils to create the different types of boundary conditions."""
+
+from typing import Optional, List
+
+import json
+
+
+class Dirichlet:
+    """Dirichlet boundary condition.
+
+    Attributes:
+        boundary_name: A string representing the boundary name of the plate
+          where the Dirichlet boundary condition will apply. The available
+          options are: left, top, right and bottom.
+        displacement_x: A float representing imposed displacement in
+          x-direction. Only fill it if want to resprtit the displamente. 
+          Fill this attribute if you want to specify the displacement.
+        displacement_y: A float representing imposed displacement in
+          y-direction. Only fill it if want to resprtit the displamente.
+          Fill this attribute if you want to specify the displacement.
+    """
+
+    def __init__(self,
+                 boundary_name: str,
```

Comment:
As per the available options, the type annotation here should be `Literal["left", "top", ...]`.